core/src/test/scala/io/qbeast/core/transform/TransformerTest.scala

```diff
@@ -40,6 +42,50 @@ class TransformerTest extends AnyFlatSpec with Matchers {
     }
   }
 
+  it should "makeTransformation with Timestamp data type" in {
+    val columnName = "a"
+    val dataType = TimestampDataType
+    val transformer = Transformer(columnName, dataType)
+
+    val minTimestamp = Timestamp.valueOf("2017-01-01 12:02:00")
+    val maxTimestamp = Timestamp.valueOf("2017-01-03 12:02:00")
+
+    val transformation = Map("a_min" -> minTimestamp, "a_max" -> maxTimestamp)
+    val resTransformation = transformer.makeTransformation(transformation)
+
+    // Find all integers in the string separated by commas and remove characters
+    val regex: Regex = "\\d+".r
+    val resMinMax = regex.findAllIn(resTransformation.toString).toList.map(_.toLong)
+    resMinMax(0) shouldBe minTimestamp.getTime
+    resMinMax(1) shouldBe maxTimestamp.getTime
+    // Get the the datatype of the transformation
+    val resDataType = resTransformation.toString.split(",")(3).dropRight(1)
+    resDataType shouldBe dataType.toString
```

Comment:
I changed the test implementation, It feels way more complicated than it should because the object does not have methods implemented to get the parameters of the class. It works but I would change it if we figure out a better way to do it.

Reply:
It's okay, but maybe you don't have to parse everything to string and use regex. You can instantiate the result transformation as a LinearTransformation, and access the corresponding methods from there.

Like:
`val resTransformation = transformer.makeTransformation(transformation).asInstanceOf[LinearTransformation]`

Reply:
Yeah, that is a much better solution